esp/esp/dbmail/models.py

```diff
@@ -355,6 +362,14 @@ class TextOfEmail(models.Model):
     send_from = models.CharField(max_length=1024) # Valid email address
     subject = models.TextField() # E-mail subject; plain text
     msgtext = models.TextField() # Message body; plain text
+
+    # Use `default` instead of `auto_now_add`, so that when a
+    # :class:`TextOfEmail` is created from a :class:`MessageRequest`, the
+    # `created_at` value may be copied over at creation time. If `auto_now_add`
+    # were `True`, the value of `now()` would always be used, even if we wanted
+    # to use something else.
+    created_at = models.DateTimeField(default=datetime.now, null=False, blank=False, editable=False, auto_now_add=False)
```

Comment:
I would suggest just not setting a default for this.  If we set a default, it will make it harder to notice error where we intended to set this manually, and didn't.  I think `TextOfEmail`s only ever get created by `MessageRequest.process`, so it's not like this will require a bunch of updating existing code.